python/hsfs/feature_group.py

```diff
@@ -1733,7 +1734,54 @@ def read_changes(
             .read(False, "default", read_options)
         )
 
-    def find_neighbors(self, embedding, col=None, k=10, filter=None, min_score=0):
+    def find_neighbors(
+        self,
+        embedding: List[Union[int, float]],
```

Comment:
We have an embedding object that we refer to as 'features' in one parameter.
Here we have the embedding vector (the array of floats, right?).
Is the naming all consistent?

Reply:
`embedding` means a vector
`EmbeddingFeature` means the feature name in vector database.
`EmbeddingIndex` means the index in vector database.